DIRECTORY.md

```
@@ -676,6 +676,8 @@
 
 ## Pystrings
   * Anagram
+    * Group Anagrams
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)